pkg/network/tracer/connection/kprobe/tcp_close_consumer.go

```diff
@@ -119,6 +128,20 @@ func (c *tcpCloseConsumer) Start(callback func([]network.ConnectionStats)) {
 				c.batchManager.GetPendingConns(oneTimeBuffer)
 				callback(oneTimeBuffer.Connections())
 				close(request)
+
+				closedCount += oneTimeBuffer.Len()
+				now := time.Now()
+				elapsed := now.Sub(then)
+				then = now
+				log.Debugf(
+					"tcp close summary: closed_count=%d elapsed=%s closed_rate=%.2f/s lost_count=%d",
```

Comment:
This is not directly related to the fix, but I've found this information extremely useful during multiple investigations, so I think we should add this log entry.